src/coreclr/gc/sample/GCSample.cpp

```diff
@@ -106,9 +112,9 @@ void WriteBarrier(Object ** dst, Object * ref)
     ErectWriteBarrier(dst, ref);
 }
 
-extern "C" HRESULT GC_Initialize(IGCToCLR* clrToGC, IGCHeap** gcHeap, IGCHandleManager** gcHandleManager, GcDacVars* gcDacVars);
+extern "C" HRESULT CALLCONV GC_Initialize(IGCToCLR* clrToGC, IGCHeap** gcHeap, IGCHandleManager** gcHandleManager, GcDacVars* gcDacVars);
 
-int __cdecl main(int argc, char* argv[])
+int CALLCONV main(int argc, char* argv[])
```

Comment:
```suggestion
int __cdecl main(int argc, char* argv[])
```
This should stay as-is. It is not unrelated to LOCALGC_CALLCONV